docs/api/browser-window.md

```diff
@@ -392,10 +389,9 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
       contain the layout of the document—without requiring scrolling. Enabling
       this will cause the `preferred-size-changed` event to be emitted on the
       `WebContents` when the preferred size changes. Default is `false`.
-  * `titleBarOverlay` Boolean (optional) -  On macOS, when using a frameless window in conjunction with
-    `win.setWindowButtonVisibility(true)` or using a `titleBarStyle` so that the traffic lights are visible,
-    this property enables the Window Controls Overlay [JavaScript APIs][overlay-javascript-apis] and
-    [CSS Environment Variables][overlay-css-env-vars].  Default is `false`.
+  * `titleBarOverlay` Object | Boolean (optional) -  When using a frameless window in conjuction with `win.setWindowButtonVisibility(true)` on macOS or using a `titleBarStyle` so that the standard window controls ("traffic lights" on macOS) are visible, this property enables the Window Controls Overlay [JavaScript APIs][overlay-javascript-apis] and [CSS Environment Variables][overlay-css-env-vars]. Default is `false`.
```

Comment:
1. Does this have any effect on Windows? The description makes it seem macOS-specific.
2. Let's explicitly state what the behavior of specifying `true` is vs. specifying an object. Maybe something like:
> Specifying `true` will result in the title bar overlay being shown with default colors.

Reply:
Yes, specifying true will result in default colors being used, similarly to not specifying an option like mentioned here. Since the concept of "default colors" only applies to Windows with no customization in macOS, maybe:

> Specifying true will result in the title bar overlay being shown with default system colors on Windows.

And maybe also modifying that description in frameless-window.md so that the same information is in both places?

Reply:
I updated both of these descriptions in the most recent commit. Are they more clear now?